### PR TITLE
Fixed cdf error.

### PR DESCRIPTION
**Pull request by mikemoss3**
Implemented a fix for the cdf method. 

Also commented out unused lines in plot methods.

These changes should address Issues #12 and #14.

### CODE REVIEW

File: asymmetric_uncertainty/core.py [hunk 1]
```diff
@@ -82,21 +82,42 @@ def pdf(self,x):
...
+                    ret[i] = 1
+                else:
+                    ret[i] = np.sum( self.pdf( x_arr[ : np.argmin(x[i] >= x_arr)] ) ) / np.sum(self.pdf(x_arr) )
+        else:
+            if x < x_arr[0]:
+                ret = 0
+            elif x > x_arr[-1]:
+                ret = 1
+            else:
+                ret = np.sum( self.pdf( x_arr[ : np.argmin(x >= x_arr)] ) ) / np.sum(self.pdf(x_arr) )
```
Comment (cgobat) [inline asymmetric_uncertainty/core.py:108]: Don't we expect `np.sum(self.pdf(x_arr))` to always be ~1?

Comment (mikemoss3) [reply asymmetric_uncertainty/core.py:108]: Yes, we absolutely do. Good find.

**Comment by cgobat** [comment]
I just tweaked the CDF implementation to streamline the code and leverage vectorized NumPy computations, and this way also works for scalar/single inputs without having to take a separate logic path. I believe the end result should be the same as in your original algorithm which checked the type and values of the input explicitly, but let me know if this looks good to you.

File: asymmetric_uncertainty/core.py (hunk 1 above)
Comment (cgobat) [reply asymmetric_uncertainty/core.py]: Actually, disregard&mdash;the $\Delta x$ spacing in `x_arr` is not necessarily 1, so even though the *integral* should always be unity, the straight sum of the $y$-values might not be.

Comment (mikemoss3) [reply asymmetric_uncertainty/core.py]: You're right that the sum of the discretized PDF won't be 1. But the sum of the PDF times the bin size will be. 
But still, isn't the CDF just the integral of PDF from (0,x). Is the sum of the entire pdf in the denominator necessary?  
If the PDF is properly normalized (which is what you showed when you took the integral in the other thread), then the CDF should need another normalization, right?

**Comment by cgobat** [comment]
Okay, I think I've gotten this to a place I'm happy with. I've done away with any attempts at numerical integration, summation, etc. The CDF is now evaluated "analytically" using a vectorized version of [`math.erf()`](https://docs.python.org/3/library/math.html#math.erf):

$$\Phi_{\mu, \sigma_\pm}(x) = \begin{cases}\frac{1}{2}\left[1+\text{erf}\left(\frac{x-\mu}{\sigma_+\sqrt{2}}\right)\right] & \text{for }x\geq\mu \\
\frac{1}{2}\left[1+\text{erf}\left(\frac{x-\mu}{\sigma_-\sqrt{2}}\right)\right] & \text{for }x<\mu \end{cases}$$

It also now handles both scalar or array inputs correctly and transparently.

**Comment by mikemoss3** [comment]
With each iteration, the MC fitting code I am using has gotten faster and faster!  

Thanks so much for working on this. I really appreciate it. It's a really useful tool.

**Pull request merged**